src/spikeinterface/widgets/unit_summary.py

```diff
@@ -30,6 +31,13 @@ class UnitSummaryWidget(BaseWidget):
     sparsity : ChannelSparsity or None, default: None
         Optional ChannelSparsity to apply.
         If SortingAnalyzer is already sparse, the argument is ignored
+    widget_params : dict or None, default: None
+        Parameters for the subwidgets in a nested dictionary
+            unitlocations_params: UnitLocationsWidget (see UnitLocationsWidget for details)
+            unitwaveforms_params: UnitWaveformsWidget (see UnitWaveformsWidget for details)
+            unitwaveformdensitymap_params : UnitWaveformDensityMapWidget (see UnitWaveformDensityMapWidget for details)
+            autocorrelograms_params : AutoCorrelogramsWidget (see AutoCorrelogramsWidget for details)
+            amplitudes_params : AmplitudesWidget (see AmplitudesWidget for details)
```

Comment:
@florian6973 I would actually drop the `_params` and add underscores (it's clear from the `widgets_params` already):
```suggestion
            unit_locations : UnitLocationsWidget (see UnitLocationsWidget for details)
            unit_waveforms : UnitWaveformsWidget (see UnitWaveformsWidget for details)
            unit_waveform_density_map : UnitWaveformDensityMapWidget (see UnitWaveformDensityMapWidget for details)
            autocorrelograms : AutoCorrelogramsWidget (see AutoCorrelogramsWidget for details)
            amplitudes : AmplitudesWidget (see AmplitudesWidget for details)
```

Reply:
done in last commit